ethstore/src/account/safe_account.rs

```diff
@@ -35,16 +36,17 @@ pub struct SafeAccount {
 	pub version: Version,
 	pub address: Address,
 	pub crypto: Crypto,
+	pub path: Option<PathBuf>,
```

Comment:
What do you think about just storing a `filename: Option<String>` instead? It will allow migrations between two `DiskDirectories` - right know accounts with `path.is_some()` can basically come from anywhere on the disk and whole `DiskDirectory` is kind of useless and misleading.